cat-launcher/src/pages/PlayPage/ReleaseNotesButton.tsx

```
@@ -0,0 +1,95 @@
+              remarkPlugins={[remarkGfm]}
+              components={{
+                a: ({ node: _node, ...props }) => (
+                  <a
+                    {...props}
+                    onClick={(e) => {
+                      if (props.href) {
+                        e.preventDefault();
+                        openLink(props.href);
+                      }
+                    }}
+                  />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)